about/index.html

```diff
@@ -113,8 +112,8 @@ <h1 class="light section-title">Features</h1>
                     <div class="card-content-text">
                         <header class="feature-card-title">Multitenancy Support</header>
                         <p>Transforming a single-tenant application into
-                            a&nbsp;multi-tenant one requires very few lines of code. You don't have to introduce
-                            <code>tenantId</code> parameters for all the calls.</p>
+                            a&nbsp;multi-tenant one requires a few lines of code. You don't have to introduce
+                            <code>tenantId</code> parameters for all the calls. <comment>Provide a link from this feature to gitHub repo or a page intended for contributors.</comment></p>
```

Comment:
Please remove the comment and have an issue.

Reply:
LGTM
Created issue #119